src/controllers/EventsController.php

```diff
@@ -353,7 +351,7 @@ function ($tag) {
 
                 // Send an email if we didn't auto-approve
                 if (! $user_mapper->isSiteAdmin($request->user_id)) {
-                    $event        = $event_mapper->getPendingEventById($event_id, true);
+                    $event        = $event_mapper->getPendingEventById($event_id);
```

Comment:
The method [`EventMapper:: getPendingEventById()`](https://github.com/joindin/joindin-api/blob/436971cedc3df8708e0e53e617ae99eb5b198a92/src/models/EventMapper.php#L1006-L1022) accepts only one parameter.